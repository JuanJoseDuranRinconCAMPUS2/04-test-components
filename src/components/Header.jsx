import React from "react";

export default function Header(){
    return (
        <>
            <div>
                <nav class = "navbar">
                    <a href='#' class = "toggle">
                        <span class="bar"></span>
                        <span class="bar"></span>
                        <span class="bar"></span>
                        <span class="bar"></span>
                    </a>
                    <div class="navbar-links">
                        <ul>
                            <li><a href='#'>Nav1</a></li>
                            <li><a href='#'>Nav2</a></li>
                            <li><a href='#'>Nav3</a></li>
                            <li><a href='#'>Nav4</a></li>
                            <li><a href='#'>Nav5</a></li>
                        </ul>
                    </div>
                </nav>
            </div>
        </>
    )
}