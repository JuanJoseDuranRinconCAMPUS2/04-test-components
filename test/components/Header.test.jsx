import { render } from "@testing-library/react";
import Header from "../../src/components/Header.jsx";

describe("Captura del componente <Header.jsx>", ()=>{
    test('Fotito :3', () => { 
        let {container} = render(<Header/>);
        expect(container).toMatchSnapshot();
    })
})
