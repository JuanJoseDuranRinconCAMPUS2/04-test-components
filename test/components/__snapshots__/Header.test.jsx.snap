// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Captura del componente <Header.jsx> Fotito :3 1`] = `
<div>
  <div>
    <nav
      class="navbar"
    >
      <a
        class="toggle"
        href="#"
      >
        <span
          class="bar"
        />
        <span
          class="bar"
        />
        <span
          class="bar"
        />
        <span
          class="bar"
        />
      </a>
      <div
        class="navbar-links"
      >
        <ul>
          <li>
            <a
              href="#"
            >
              Nav1
            </a>
          </li>
          <li>
            <a
              href="#"
            >
              Nav2
            </a>
          </li>
          <li>
            <a
              href="#"
            >
              Nav3
            </a>
          </li>
          <li>
            <a
              href="#"
            >
              Nav4
            </a>
          </li>
          <li>
            <a
              href="#"
            >
              Nav5
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </div>
</div>
`;
